media-video/gaupol/metadata.xml

```diff
@@ -9,9 +9,12 @@
 		<email>media-video@gentoo.org</email>
 		<name>Gentoo Video project</name>
 	</maintainer>
-	<stabilize-allarches/>
+	<use>
+		<flag name="player">Enables preview support with an integrated player</flag>
+	</use>
 	<upstream>
 		<remote-id type="github">otsaloma/gaupol</remote-id>
 		<bugs-to>https://github.com/otsaloma/gaupol/issues</bugs-to>
 	</upstream>
+	<stabilize-allarches/>
```

Comment:
Accidentally left in?

Reply:
You added it :P

Reply:
Oh, so its that you just moved it for some reason.

Reply:
>  I see no reason for it being there. Should I remove it @sbraz?

Hi, what do you mean by that? gaupol is a pure-Python package and it was added by @thesamesam ine1af9fe1721d5e6b1dc42cf891c988786d289cbd.

Reply:
glibc is also architecture-dependent. Ergo, ALLARCHES can't exist.